nipype/pipeline/plugins/multiproc.py

```diff
@@ -217,32 +178,38 @@ def _send_procs_to_workers(self, updatehash=False, graph=None):
         executing_now = []
 
         # Check to see if a job is available
-        currently_running_jobids = np.flatnonzero((self.proc_pending == True) & \
-                                (self.depidx.sum(axis=0) == 0).__array__())
+        currently_running_jobids = np.flatnonzero(
+            self.proc_pending & (self.depidx.sum(axis=0) == 0).__array__())
```

Comment:
What work is `__array__` doing here?

Reply:
I didn't want to mess much around these checks since they work. But yeah, I don't get it either. I'll clean up.